ui/decorators.py

```diff
@@ -0,0 +1,27 @@
+"""
+Decorators for views
+"""
+from functools import wraps
+
+from django.shortcuts import redirect
+
+
+def require_terms_of_service(f):
+    """
+    If user profile does not have terms of service, redirect to terms of service
+    """
+    @wraps(f)
+    def wrapper(request, *args, **kwargs):
+        """
+        Wrapper for require_terms_of_service
+
+        Args:
+            request (django.http.request.HttpRequest): A request
+        """
+        if not request.user.is_anonymous() and request.path != '/terms_of_service/':
```

Comment:
`'/terms_of_service/'` should probably go in a variable: at least a global one inside this module